bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/internal/validation/MinMaxValidator.java

```diff
@@ -37,9 +39,11 @@ final class MinMaxValidator implements ConfigDescriptionParameterValidator {
             return null;
         }
 
-        // Allow specified options to be outside of the min/max value
+        Object normalizedValue = ConfigUtil.normalizeType(value, parameter).toString();
```

Comment:
It technically doesn't matter, but make this `String normalizedValue = ...` for clarity?